update/common/scripts/5.4/updatepasswordhashlength.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
What's the use of the script if we have upgrade SQL scripts?

Reply:
I guess we don't need the script, the sql however should be updated to reflect the release it will be in 2017.10 _(if it makes it for that)_.